apps/console/src/features/private-key-jwt/pages/private-key-jwt-config.tsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import { Section } from "@wso2is/react-components";
+import React, { FunctionComponent, ReactElement, useEffect } from "react";
+import { useTranslation } from "react-i18next";
+import { Divider } from "semantic-ui-react";
+import { AppConstants, history } from "../../core";
+import { getSettingsSectionIcons } from "../../server-configurations";
+import { useTokenReuseConfigData } from "../api";
+
+/**
+ * Props for PrivateKey JWT Config Interface settings page.
+ */
+type PrivateKeyJWTConfigPageInterface = IdentifiableComponentInterface;
+
+/**
+ * PrivateKey JWT connector listing page.
+ *
+ * @param props - Props injected to the component.
+ * @returns PrivateKey JWT connector listing page component.
+ */
+export const PrivateKeyJWTConfig: FunctionComponent<PrivateKeyJWTConfigPageInterface> = (
+    props: PrivateKeyJWTConfigPageInterface
+): ReactElement => {
+    const { [ "data-componentid" ]: componentId } = props;
+    const { t } = useTranslation();
+
+    const {
+        data: tokenReuseData,
+        isLoading: isLoading
+    } = useTokenReuseConfigData();
+
+    useEffect(() => {
+        if (isLoading) {
+            return;
+        }
+    }, [ isLoading ]);
+
+    /**
+     * Handle connector advance setting selection.
+     */
+    const handleSelection = () => {
+        history.push(AppConstants.getPaths().get("PRIVATE_KEY_JWT_CONFIG_EDIT"));
+    };
+
+    return ( tokenReuseData ?
```

Comment:
Check if we can use `&&`

Reply:
I think ? is enough because tokenReuseData is a conditional object. Hence I check for the existence and return the DOM

Reply:
Suggestion is to optimize

```
tokenReuseData 
&& (
    <Section>
    ----
    </Section>
)
```

Reply:
Using && for conditional rendering is an anti-pattern.   

Ref: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx